tests/test-neve-dynamic-selector.php

```diff
@@ -285,8 +285,8 @@ public function test_css_generation_with_extra_selectors() {
 		] );
 		$css        = '' . $subscriber;
 
-		$this->assertContains( ",.one-test-selector", $css );
-		$this->assertNotContains( \Neve\Core\Settings\Config::CSS_SELECTOR_BTN_PRIMARY_HOVER, $css );
-		$this->assertContains( "color: #fff", $css );
+		$this->assertStringContainsString( ",.one-test-selector", $css );
+		$this->assertStringContainsString( \Neve\Core\Settings\Config::$css_selectors_map[\Neve\Core\Settings\Config::CSS_SELECTOR_BTN_PRIMARY_HOVER], $css );
```

Comment:
```suggestion
		$this-> assertStringNotContainsString( \Neve\Core\Settings\Config::CSS_SELECTOR_BTN_PRIMARY_HOVER, $css );
```
Isn't the correct usage to use `assertStringNotContainsString` as I see previously it was `assertNotContains`? Am I wrong? I believe the test was there to ensure that it will return already the replaced CSS selector so `buttons_primary_hover` string should not be there.

Reply:
You are right. I accidentally created another test. I will re-add the old one 🙌